api/v1beta1/kuadrant_types.go

```diff
@@ -29,6 +31,26 @@ import (
 
 // KuadrantSpec defines the desired state of Kuadrant
 type KuadrantSpec struct {
+	// +optional
+	Limitador *LimitadorSpec `json:"limitador,omitempty"`
```

Comment:
I suppose we don't want the entire [`limitadorv1alpha1.LimitadorSpec`](https://github.com/Kuadrant/limitador-operator/blob/e0517a252dd7a96fa117adfb443e356a6233eb6d/api/v1alpha1/limitador_types.go#L61) type here, right?

Reply:
Correct. Some spec may be forced by the operator and not user editable. While others are classified as dev/test and should not be easily set in the kuadrant operator.